src/Tribe/Tickets_Handler.php

```diff
@@ -124,6 +124,36 @@ public function print_checkedin_totals() {
 	}
 
 	/**
+	 * Get a count of all the tickets for an event. Queries all active modules/providers.
+	 *
+	 * @static
+	 *
+	 * @param $event_id
+	 *
+	 * @return int number of tickets ( 0 means unlimited )
+	 */
+	public function get_event_tickets_count() {
+		global $post;
+		$tickets_count = 0;
+
+		$tickets = Tribe__Tickets__Tickets::get_event_tickets( $post->ID );
+
+		if ( ! empty( $tickets ) ) {
+			foreach ( $tickets as $ticket ) {
+				$stock = $ticket->original_stock();
+
+				if ( empty( $stock ) ) {
+					return -1;
```

Comment:
Sorry, this needs a comment - that's "unlimited". I'm going to rename this function a bit more clearly and add a comment.